control/trajectory_follower_node/param/test.param.yaml

```diff
@@ -0,0 +1,12 @@
+/**:
+  ros__parameters:
+    lateral_controller_mode: "mpc"
+    longitudinal_controller_mode: "pid"
+
+    # steer offset
+    steering_offset:
```

Comment:
These parameters must be in the lateral_controller param as in autoware_launch.
https://github.com/autowarefoundation/autoware_launch/blob/main/autoware_launch/config/control/trajectory_follower/lateral/mpc.param.yaml#L66-L72

Would you move them into [here](https://github.com/autowarefoundation/autoware.universe/blob/9983ead76b3fc164f23aab6d8fe1d27519f1770c/control/trajectory_follower_node/param/lateral_controller_defaults.param.yaml)?

Reply:
Check my latest comment, whether it resolves your issue.